cli.js

```diff
@@ -6,6 +6,7 @@ commander
 //  .version(meta.version)
   .usage('[options] <endpoint>')
   .option('-p, --port <n>', 'Local proxy port, default 9200', parseInt)
+  .option('-p, --bindIf <ip>', 'Bind to interface, defaults to 127.0.0.1', /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/, '127.0.0.1')
```

Comment:
I assume shorthand parameter should be `-b` and not `-p` since thats already used.